.github/workflows/copy-workflow.yml

```diff
@@ -23,14 +23,33 @@ jobs:
         repository: ${{ matrix.cfg.target }}
         token: ${{ secrets.WEB3BOT_GITHUB_TOKEN }}
         persist-credentials: true
+    - name: git config
+      run: |
+        git config user.name web3-bot
+        git config user.email "web3-bot@users.noreply.github.com"
+    - name: run go mod tidy
+      run: |
+        go mod edit -go 1.15
+        go mod tidy
+        if ! git diff --quiet; then
+          git add .
+          git commit -m "run go mod tidy"
```

Comment:
it might be confusing to a user that the go version was changed, which is not something that tidy would do by itself.

how about "set go version to 1.15 and go mod tidy"?

Reply:
Done.